### PR TITLE
Chore: Add markdown linting

### LINKED ISSUE #22: [CHORE] - Add Testing
### What work needs to be performed?

We should add `markdownlint` and `markdownlint-cli` to test this repo.

### Additional information

Refer to https://github.com/naomi-lgbt/notes for implementation idea.

### PR DESCRIPTION
**Pull request by a2937**
# Pull Request

## Description:

It adds markdown lint and markdown lint CLI to this project. There are only three relevant issues to deal with, all dealing with no bare URLs in the code of conduct. Can docsify handle URLs being written like `<https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>`. If so, I will proceed to correct it. And unfortunately, it is complaining about a violation in the navbar which isn't technically a violation because it is a fragment of a page. I could set it to scan the pages directory only if you'd like. 

## Related Issue:

Closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +25,9 @@
   "homepage": "https://github.com/carrigan-community/handbook#readme",
   "dependencies": {
     "docsify-cli": "4.4.4"
+  },
+  "devDependencies": {
+    "markdownlint": "^0.28.1",
+    "markdownlint-cli": "^0.33.0"
   }
```
Comment (naomi-lgbt) [inline package.json:32]:
```suggestion
  "devDependencies": {
    "markdownlint": "0.28.1",
    "markdownlint-cli": "0.33.0"
  }
```
These will need to be pinned and the `package-lock.json` will need to be synced (`npm i` + commit and push)

File: .markdownlint.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "default": true,
+  "MD013": false
```
Comment (naomi-lgbt) [inline .markdownlint.json:3]:
```suggestion
  "line-length": false
```

Use the human-readable name for maintainability

File: package.json (hunk 1 above)
Comment (a2937) [reply package.json]: It was just fixed.

File: .markdownlint.json (hunk 2 above)
Comment (a2937) [reply .markdownlint.json]: Fixed. And thank you. That's a lot easier to read.

**Review by naomi-lgbt — APPROVED** [review]
Good to go!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**